simulation_parameters/Constants.cs

```diff
@@ -183,12 +183,38 @@ public static class Constants
     /// </remarks>
     public const float BASE_MOVEMENT_ATP_COST = 1.0f;
 
+    public const float BASE_MOVEMENT_FORCE = 900.0f;
+
+    public const float MAX_STRAIN_PER_CELL = 100.0f;
+
+    public const float PASSIVE_STRAIN_DECREASE_PER_UPDATE = 0.5f;
```

Comment:
Rather than per update rate, any change rates should be scaled to be per second.